odata-csdl/1 Introduction.md

```diff
@@ -26,6 +26,18 @@ Schema Definition Language (XSD) 1.1 as described in
 
 ## ##subsec Changes from Earlier Versions
 
+Section | Feature / Change | OData Issue
```

Comment:
```suggestion
Section | Feature / Change | OData Issue | GitHub commit containing the change
```
Add a column linking to the commit diff view

Reply:
How do I do this if I have multiple commits (for example due to reviews) and haven't yet squash-merged the PR?

Should we rather link to the PR?

Reply:
Well, actually the Jira issue already links to the pull request(s), so this would be somewhat redundant.

Reply:
> How do I do this if I have multiple commits (for example due to reviews) and haven't yet squash-merged the PR?

I suggest to add the link only after the PR is merged. This happens in a separate PR that will be merged immediately before the document is sent to public review.

Reply:
Then rather link to the PRs, which also provides the diff view, or just link to the issue which has the PR link.